scripts/screens/WarriorDenScreen.py

```diff
@@ -322,7 +320,7 @@ def create_top_info(self):
 
         last_change_text = "unknown"
         next_change = "0 moons"
-        if game.clan.last_focus_change:
+        if game.clan.last_focus_change == 0 or game.clan.last_focus_change:
```

Comment:
could we also consider:

```suggestion
        if isinstance(game.clan.last_focus_change, int):
```

IMO, this is much clearer than the previous, since it appears to imply that `game.clan.last_focus_change` can either be `0` or `True`, which is... confusing.